### ISSUE TITLE
Add selector validation

### ISSUE THREAD

**Issue by dgzlopes** [grafana/xk6-disruptor#41]
Right now, we don't validate that the selector the user passes is correct. 

I spent some time trying to use the following selector:
```js
const selector = {
	namespace: 'k6-cloud-test',
	labels: {
		name: 'querier',
	},
}
```

And it didn't work as I expected! It matched only the namespace and omitted the labels as the schema was wrong. 

We should point that out to the users.

**Comment by pablochacin** [comment]
This issue is related to #45.

**Comment by pablochacin** [comment]
Fixed by #93

**Comment by pablochacin** [comment]
Fixed by https://github.com/grafana/xk6-disruptor/issues/93
